pkg/apis/config/v1beta1/defaults_test.go

```diff
@@ -349,7 +349,7 @@ func newFakeSuggestionConfig(algorithmName string) *SuggestionConfig {
 			AccessModes: []corev1.PersistentVolumeAccessMode{
 				DefaultSuggestionVolumeAccessMode,
 			},
-			Resources: corev1.ResourceRequirements{
+			Resources: corev1.VolumeResourceRequirements{
```

Comment:
Dedicated API is introduced into the PVC: https://github.com/kubernetes/kubernetes/pull/118653